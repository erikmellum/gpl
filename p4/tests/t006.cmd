-s 42 tests/t006.gpl
