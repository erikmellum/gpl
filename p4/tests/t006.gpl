// p4 t006.gpl
int a_very_long_identifier;
int an_identifier_with_digits_1234;
int _start_with_an_underscore;
