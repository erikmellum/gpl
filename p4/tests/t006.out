gpl.cpp::main()
  input file(tests/t006.gpl)
  random seed(42)
  read_keypresses_from_standard_input(false)
  dump_pixels(false)
  symbol_table(true)
  print_symbol_table(true)
  graphics(false)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().

No errors found (parser probably worked correctly).


Printing the symbol table from main()
int _start_with_an_underscore 42
int a_very_long_identifier 42
int an_identifier_with_digits_1234 42
If graphics was enabled gpl program would execute now.

gpl.cpp::main() done.
